Wrappers/Python/cil/io/RAWFileWriter.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+#   This work is part of the Core Imaging Library (CIL) developed by CCPi 
+#   (Collaborative Computational Project in Tomographic Imaging), with 
+#   substantial contributions by UKRI-STFC and University of Manchester.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.import numpy as np
+
+from cil.framework import AcquisitionData, ImageData
+import os, re
+import sys
+from cil.framework import AcquisitionData, ImageData
+import re
+from cil.io import utilities
+import configparser
+
+import logging
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+def compress_and_save(data, compress, scale, offset, dtype, fname):
+    '''Compress and save numpy array to file
+    
+    Parameters
+    ----------
+    data : numpy array
+    compress : bool
+    scale : float
+    offset : float
+    dtype : numpy dtype
+    fname : string
+
+    Note:
+    -----
+    Data is always written in ‘C’ order, independent of the order of d.
+    '''
+    if compress:
+        d = utilities.compress_data(data, scale, offset, dtype)
+    else:
+        d = data
+        
+    logging.info("Data is always written in ‘C’ order, independent of the order of d.")
```

Comment:
ah you have used `logging` and not `logger` here, that's correct. So the logger set up is probably redundant.

Reply:
yes